### PR TITLE
Feat/trend query complete

### PR DESCRIPTION
**Pull request by epan**
Adds

1. Functional trendQuery method that will get Google Trends for a keyword and convert into desired format
2. Add stub test for integrating this (need to figure out how to best simulate network call and dynamic data)

### CODE REVIEW

File: utilities/trendQuery.js [hunk 1]
```diff
@@ -1,15 +1,16 @@
...
   const options = {
     keyword: keyword,
-    startTime: backDateByMonth(15),
+    startTime: backDateByMonths(15),
   };
 
   googleTrends.interestOverTime(options)
   .then((results) => {
-    callback(results);
+    callback(sanitizeTrend(JSON.parse(results)));
```
Comment (aseiden) [inline utilities/trendQuery.js]: Minor comment, no need to change prior to pull.  For sep of concerns I would stick the parse JSON in the sanitizeTrend helper.  No need to recommit over that though.

**Review by aseiden — DISMISSED** [review]
lgtm - 1 minor comment

File: utilities/trendQuery.js (hunk 1 above)
Comment (epan) [reply utilities/trendQuery.js]: @aseiden I thought about that too. I guess I was wondering if we should assume that the `googleTrends.interestOvertime` returns an object but it is already designed the way it is and it gives a string so I am actually 100% for moving the parse to the `sanitizeTrend` so I'll do that in re-submit 👯‍♂️

**Comment by epan** [comment]
ugh darnit. I rebased from upstream master then couldnt push to my remote branch then did a pull and i think this PR is assuming I'm committing stuff fufei already made 🦀

**Review by aseiden — APPROVED** [review]
lgtm

**Pull request merged**